mailparser/mailparser.py

```diff
@@ -226,8 +226,12 @@ def message_id(self):
 
     @property
     def to_(self):
-        return decode_header_part(
-            self._message.get('to', self._message.get('delivered-to')))
+        to_ = self._message.get('to')        
+        if not to_:
+            return None
+        else:
```

Comment:
This is from the Enron email dataset (arnold-j,_sent_mail, #358). 
I thought I just modified the code to handle the case where there was no 'To:' field to parse. It seems to work and makes sense to me I think--what did I do incorrectly? 

Other headers have both the X-to and To fields, which aren't always the same. Meaning the X-to is not reliable enough for me to worry about parsing out.
E.g.
Message-ID: <32572339.1075857602061.JavaMail.evans@thyme>
Date: Tue, 11 Apr 2000 09:58:00 -0700 (PDT)
From: john.arnold@enron.com
To: mike.maggi@enron.com
Subject: Option Analysis on NG Price Book
Mime-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Transfer-Encoding: 7bit
X-From: John Arnold
X-To: Mike Maggi
X-cc: 
X-bcc: 
X-Folder: \John_Arnold_Dec2000\Notes Folders\'sent mail
X-Origin: Arnold-J
X-FileName: Jarnold.nsf